recipes/ecmwf-api-client/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install
+
+# Add more build steps here, if they are necessary.
+
+# See
+# http://docs.continuum.io/conda/build.html
+# for a list of environment variables that are set during the build process.
```

Comment:
Hi @Ohjeah you don't need the `bld.bat` nor the `build.sh` here. (See https://github.com/conda-forge/staged-recipes/pull/685/files#r64913933)